lib/modules/dosomething/dosomething_northstar/dosomething_northstar.auth.inc

```diff
@@ -151,8 +151,12 @@ function dosomething_northstar_set_authorization_action($action, $nid, $params =
     return;
   }
 
-  $_SESSION['openid_connect_action'] = ['signup' => $nid];
-  $_SESSION['affiliate_messaging_opt_in'] = isset($params['affiliate_messaging_opt_in']) ? $params['affiliate_messaging_opt_in'] : FALSE;
+  $_SESSION['openid_connect_action'] = [
+    'signup' => $nid,
+    'params' => [
```

Comment:
Should this be part of the `signup` "action", since it doesn't do anything on it's own? It feels awkward to loop over these but then "grab" this one on the `signup` iteration.